<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QWidget" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>280</width>
     <height>120</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>        sin</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="sin_edit">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btn1">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="result1">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>   arcsin</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="arcsin_edit">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btn2">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="result2">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>       cos</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="cos_edit">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btn3">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="result3">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>  arctan</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="arccos_edit">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="btn4">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="result4">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
